README.md

```diff
@@ -15,8 +15,7 @@ $ git clone https://github.com/acid-info/logos-docs-template.git
 $ yarn install
 ```
 
-3. Create .env and run the scraping script
-and 
+3. Create .env and run the scraping script (make sure you're located in the root of the project when running this command)
```

Comment:
That's because the repo may contain some folders that are not specification related (assets, configs etc.), we don't want to parse those, so we basically have a whitelist.
As to why it's part of the env and not hard coded, it's because we don't want to be doing commits to the codebase that are environment related.

Reply:
I don't think I understand. How is list of sites to scrape "environment related"? Seems like part of a build static configuration that should not and doesn't need to be a variable config.

Reply:
Well, locally when testing, you might not need to build the whole site but just the parts which had new updates